frontend/src/entry/model/InferenceServices.tsx

```diff
@@ -27,6 +55,41 @@ export default function InferenceServices({ model }: InferenceProps) {
     [model, inferences],
   )
 
+  const handleCreateToken = async () => {
+    setErrorMessage('')
+    const error = MultipleErrorWrapper('Unable to create an authenication token', { isTokensError, isUiConfigError })
+    if (error) return error
+    const authorizationTokenName = uiConfig?.inference.authorizationTokenName
+    const authorizationAccessKeys = tokens.filter((value) => value.description === authorizationTokenName)
+    if (authorizationTokenName && authorizationAccessKeys) {
+      for (const token of authorizationAccessKeys) {
+        const response = await deleteUserToken(token.accessKey)
+
+        if (!response.ok) {
+          setErrorMessage(await getErrorMessage(response))
```

Comment:
I would think we'd want to stop processing any more tokens if this has failed. Consider doing:
```
if (!response.ok) {
    setErrorMessage(await getErrorMessage(response))
    mutateTokens()
    return
}
```